### ISSUE TITLE
(3.0.60) 'Browse Class' on Transcript gets walkback

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#429]
The method source pane has a pop-up menu option ``Browse Class`` which appears when a class is highlighted. However, it also appears when ``Transcript`` is selected. Pressing the menu item gives a walkback in this case. 

Either we should tighten up when we offer that menu item, or give a better error when the selected "class" is not a class. 

```
a MessageNotUnderstood occurred (error 2010), a TranscriptStreamPortable does not understand  #'superClass'
MessageNotUnderstood >> defaultAction @2 line 3
MessageNotUnderstood (AbstractException) >> _signalWith: @5 line 25
MessageNotUnderstood (AbstractException) >> signal @2 line 47
TranscriptStreamPortable (Object) >> doesNotUnderstand: @9 line 10
TranscriptStreamPortable (Object) >> _doesNotUnderstand:args:envId:reason: @7 line 12
RowanClassService >> basicRefreshFrom: @4 line 5
RowanClassService >> refreshFrom: @2 line 3
RowanClassService >> updateClass @8 line 9
RowanClassService >> update @2 line 3
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @6 line 11
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1
```

**Comment by LisaAlmarode** [comment]
It's a general problem with non-class Globals.  Same thing occurs for PlusInfinity.

**Comment by ericwinger** [comment]
Fixed in 3.0.63. https://github.com/GemTalk/Jadeite/commit/bd43759df010ee500f2e9fcbb0d9520b4648d72f
If a non-class is browsedin the source pane, the class of the object is browsed. i.e.  PlusInfinity browses Float.

**Comment by LisaAlmarode** [comment]
There are two problems left.

 - [x] The transcript no longer gives me any option to browse class when a class or a non-class global is selected.  So the bug is fixed (and was in v.62 for that matter), but I think it was a handy feature there.

The source pane of the project browser nicely does the right thing for kernal classes and Globals like PlusInfinity (open a dictionary browser), However,

 - [x] if you browse class on a rowanized class, it uses the current project browser so you lose what you're doing.  It should open a new project browser in this case.

**Comment by dalehenrich** [comment]
I use tabs a lot in the project browser, so I would prefer that I get a new tab in the project browser ... this might only make sense if you are doing a find class from the project browser although it would be cool if Jadeite to add a tab to an existing project browser

**Comment by ericwinger** [comment]
@LisaAlmarode  I'm confused by check box one.  I see an option to Browse Class when Transcript is selected. 
![image](https://user-images.githubusercontent.com/36212516/53131261-168c5800-3521-11e9-80f7-a205a2d6e903.png)

**Comment by ericwinger** [comment]
Second check box fixed in https://github.com/GemTalk/Jadeite/commit/df00723d045102f0f245b49ba3c0b4e5bac52bd2
New browser is opened if rowanized class is selected in both method & project browser. Holding ctrl opens new tab instead (from within project browser only)

**Comment by LisaAlmarode** [comment]
"transcript" may have been confusing to one or the other of us, since the problem is the console window which I automatically could consider as "the transcript"... you know, the window that you get when you first log in...   I believe that used to let you browse a class but doesn't now.

**Comment by ericwinger** [comment]
I see now. Not sure that the transcript ever let you browse classes, implementors, or senders but it (and the workspace) does now. Fixed in 3.0.64. https://github.com/GemTalk/Jadeite/commit/407a65b309e46d7a33845c891a43335cf3d93f88

**Comment by LisaAlmarode** [comment]
The control-browse class is a nice feature.  Seems hard to document such that it doesn't get lost.
